test/unit/PositionHouse_02.spec.ts

```diff
@@ -4979,5 +4979,135 @@ describe("PositionHouse_02", () => {
             console.log("listOrderPending2", listOrderPending2[0].partialFilled.toString())
             await expect(listOrderPending2[0].partialFilled.toString()).eq("5")
         })
+
+        it("should uninitialize pip when cancel the last order in pip", async () => {
```

Comment:
unit test for cancel only the last order and all of the pending orders in current pip